doc/source/whatsnew/v0.25.0.rst

```diff
@@ -649,6 +649,7 @@ I/O
 - Bug in :func:`read_json` where date strings with ``Z`` were not converted to a UTC timezone (:issue:`26168`)
 - Added ``cache_dates=True`` parameter to :meth:`read_csv`, which allows to cache unique dates when they are parsed (:issue:`25990`)
 - :meth:`DataFrame.to_excel` now raises a ``ValueError`` when the caller's dimensions exceed the limitations of Excel (:issue:`26051`)
+- Bug while selecting from :class: `pandas.HDFstore` with `where=''` specified (:issue:`26610`).
```

Comment:
no space in ``:class:`HDFStore` `` (also spelling), use double-backticks around ``where``

Reply:
Done. Check please. Sorry - have never worked with `.rst` before.